File: YaroWilliams/Exercise6.java
import java.util.Scanner;
public class Example6 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Enter your height in m");
        double height = input.nextDouble();
        System.out.println("Enter your weight in kg");
        double weight = input.nextDouble();

        int BMI = (int)(weight / (height * height));
        System.out.println("Your Body Mass Index is: " + BMI);
    }
}

File: Yusif/Day2-Challenges/Exercise6.js
const { parse } = require("path");
const readline =require("readline")

const rl=readline.createInterface({
    input:process.stdin,
    output:process.stdout
});
rl.question("Enter your height in m: ",(height)=>{
    rl.question("Enter your weight in kg: ",(weight)=>{
        const w=parseInt(weight)
        const h=parseInt(height)
        const BMI=Math.round(w/(h*h));
        console.log(`your Body Mass Index is: ${BMI}`)
        rl.close()

    })

}
)